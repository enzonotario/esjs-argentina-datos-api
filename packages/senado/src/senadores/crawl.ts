import * as cheerio from 'cheerio'
import { writeEndpoint } from '../utils/writeEndpoint.ts'

export interface Senador {
  foto: string
  nombre: string
  provincia: string
  partido: string
  email: string
  telefono: string
  redes: string[]
}

export async function crawl(): Promise<Senador[]> {
  const response = await fetch(
    'https://www.senado.gob.ar/senadores/listados/listaSenadoRes',
  )

  const html = await response.text()

  const $ = cheerio.load(html)

  const senadores: Senador[] = []

  $('tr').each((_, el) => {
    const $el = $(el)
    const foto = `https://www.senado.gob.ar${$el.find('img').attr('data-src')}`
    const nombre = $el.find('a').eq(1).text().trim().replace(/\s+/g, ' ')
    const provincia = $el.find('td').eq(2).text().trim()
    const partido = $el.find('td').eq(3).text().trim()
    const email = $el.find('li').eq(0).text().trim()
    const telefono = $el.find('li').eq(1).text().trim()
    const redes = [
      ...$el.find('li').map((_, el) => {
        return String($(el).find('a').attr('href'))
          .trim()
          .replace(/^mailto:/, '')
      }),
    ]
      .filter(Boolean)
      .filter(red => red !== 'undefined')

    if (!nombre) {
      return
    }

    const senador: Senador = {
      foto,
      nombre,
      provincia,
      partido,
      email,
      telefono,
      redes,
    }

    senadores.push(senador)
  })

  writeEndpoint('/senadores', senadores)

  return senadores
}
