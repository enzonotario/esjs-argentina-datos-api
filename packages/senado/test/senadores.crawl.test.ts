import { describe, expect, it } from 'vitest'
import { crawl } from '../src/senadores/crawl'

describe(
  'actas',
  () => {
    it('crawl', async () => {
      const result = await crawl()

      console.log({result})

      expect(result).toMatchSnapshot()
    })
  },
  {
    timeout: 300000,
  },
)
