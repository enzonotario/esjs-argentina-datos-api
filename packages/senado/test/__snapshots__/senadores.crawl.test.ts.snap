// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`actas > crawl 1`] = `
[
  {
    "email": "maximiliano.abad@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/546.gif",
    "nombre": "Abad, Maximiliano",
    "partido": "Juntos por el Cambio",
    "provincia": "Buenos Aires",
    "redes": [
      "maximiliano.abad@senado.gob.ar",
      "https://www.facebook.com/maxiabad?mibextid=9R9pXO",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "bartolome.abdala@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/564.gif",
    "nombre": "Abdala, Bartolomé Esteban",
    "partido": "Alianza la Libertad Avanza",
    "provincia": "San Luis",
    "redes": [
      "bartolome.abdala@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "carmen.alvarez@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/522.gif",
    "nombre": "Alvarez Rivero, Carmen",
    "partido": "Juntos por el Cambio",
    "provincia": "Córdoba",
    "redes": [
      "carmen.alvarez@senado.gob.ar",
      "https://www.facebook.com/CarmenAlvarezRiveroOficial",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "guillermo.andrada@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/519.gif",
    "nombre": "Andrada, Guillermo Eduardo",
    "partido": "Frente de Todos",
    "provincia": "Catamarca",
    "redes": [
      "guillermo.andrada@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "carlos.arce@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/558.gif",
    "nombre": "Arce, Carlos Omar",
    "partido": "Frente Renovador de la Concordia-Innovación Federal",
    "provincia": "Misiones",
    "redes": [
      "carlos.arce@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "ivanna.arrascaeta@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/565.gif",
    "nombre": "Arrascaeta, Ivanna Marcela",
    "partido": "Alianza la Libertad Avanza",
    "provincia": "San Luis",
    "redes": [
      "ivanna.arrascaeta@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "ezequiel.atauche@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/552.gif",
    "nombre": "Atauche, Ezequiel",
    "partido": "Partido Renovador Federal",
    "provincia": "Jujuy",
    "redes": [
      "ezequiel.atauche@senado.gob.ar",
      "https://www.facebook.com/Ezequiel.Atauche",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "beatriz.avila@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/541.gif",
    "nombre": "Avila, Beatriz Luisa",
    "partido": "Juntos por el Cambio",
    "provincia": "Tucumán",
    "redes": [
      "beatriz.avila@senado.gob.ar",
      "http://www.facebook.com/beatrizavilaok",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "vilma.bedia@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/553.gif",
    "nombre": "Bedia, Vilma Facunda",
    "partido": "Partido Renovador Federal",
    "provincia": "Jujuy",
    "redes": [
      "vilma.bedia@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "daniel.bensusan@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/532.gif",
    "nombre": "Bensusán, Daniel Pablo",
    "partido": "Frente de Todos",
    "provincia": "La Pampa",
    "redes": [
      "daniel.bensusan@senado.gob.ar",
      "https://www.facebook.com/dpbensusan",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "pablo.blanco@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/511.gif",
    "nombre": "Blanco, Pablo Daniel",
    "partido": "Juntos por el Cambio Tierra Del Fuego",
    "provincia": "Tierra del Fuego, Antártida e Islas del Atlántico Sur",
    "redes": [
      "pablo.blanco@senado.gob.ar",
      "https://www.facebook.com/pablodanielblancosenador",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "jose.carambia@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/567.gif",
    "nombre": "Carambia, José María",
    "partido": "Alianza por Santa Cruz",
    "provincia": "Santa Cruz",
    "redes": [
      "jose.carambia@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "lucia.corpacci@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/398.gif",
    "nombre": "Corpacci, Lucia Benigna",
    "partido": "Frente de Todos",
    "provincia": "Catamarca",
    "redes": [
      "lucia.corpacci@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "lucila.crexell@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/440.gif",
    "nombre": "Crexell, Carmen Lucila",
    "partido": "Juntos por el Cambio",
    "provincia": "Neuquén",
    "redes": [
      "lucila.crexell@senado.gob.ar",
      "https://twitter.com/LucilaCrexell",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "andrea.cristina@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/570.gif",
    "nombre": "Cristina, Andrea Marcela",
    "partido": "Juntos por el Cambio Chubut",
    "provincia": "Chubut",
    "redes": [
      "andrea.cristina@senado.gob.ar",
      "https://www.facebook.com/AndreaCristina.Chubut?mibextid=LQQJ4d",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "alfredo.deangeli@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/438.gif",
    "nombre": "de Angeli, Alfredo Luis",
    "partido": "Juntos por el Cambio",
    "provincia": "Entre Ríos",
    "redes": [
      "alfredo.deangeli@senado.gob.ar",
      "https://twitter.com/alfredodeangeli",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "eduardo.depedro@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/545.gif",
    "nombre": "de Pedro, Eduardo Enrique",
    "partido": "Alianza Unión por la Patria",
    "provincia": "Buenos Aires",
    "redes": [
      "eduardo.depedro@senado.gob.ar",
      "https://www.facebook.com/wadodecorrido",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "juliana.ditullio@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/517.gif",
    "nombre": "di Tullio, Juliana",
    "partido": "Alianza Unión por la Patria",
    "provincia": "Buenos Aires",
    "redes": [
      "juliana.ditullio@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "martin.donate@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/505.gif",
    "nombre": "Doñate, Claudio Martín",
    "partido": "Frente de Todos",
    "provincia": "Río Negro",
    "redes": [
      "martin.donate@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "maria.dure@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/512.gif",
    "nombre": "Duré, María Eugenia",
    "partido": "Frente de Todos",
    "provincia": "Tierra del Fuego, Antártida e Islas del Atlántico Sur",
    "redes": [
      "maria.dure@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "carlos.espinola@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/461.gif",
    "nombre": "Espínola, Carlos Mauricio",
    "partido": "Frente Todos",
    "provincia": "Corrientes",
    "redes": [
      "carlos.espinola@senado.gob.ar",
      "https://twitter.com/espinolacamau",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "flavio.fama@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/520.gif",
    "nombre": "Fama, Flavio Sergio",
    "partido": "Juntos por el Cambio",
    "provincia": "Catamarca",
    "redes": [
      "flavio.fama@senado.gob.ar",
      "https://www.facebook.com/flaviofamaoficial",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "anabel.fernandezsagasti@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/468.gif",
    "nombre": "Fernández Sagasti, Anabel",
    "partido": "Frente de Todos",
    "provincia": "Mendoza",
    "redes": [
      "anabel.fernandezsagasti@senado.gob.ar",
      "https://www.facebook.com/anabelfsagasti",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "natalia.gadano@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/568.gif",
    "nombre": "Gadano, Natalia Elena",
    "partido": "Alianza por Santa Cruz",
    "provincia": "Santa Cruz",
    "redes": [
      "natalia.gadano@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "eduardo.galaretto@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/573.gif",
    "nombre": "Galaretto, Eduardo Horacio",
    "partido": "Juntos por el Cambio",
    "provincia": "Santa Fe",
    "redes": [
      "eduardo.galaretto@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "silvina.garcialarraburu@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/441.gif",
    "nombre": "García Larraburu, Silvina Marcela",
    "partido": "Frente de Todos",
    "provincia": "Río Negro",
    "redes": [
      "silvina.garcialarraburu@senado.gob.ar",
      "https://www.facebook.com/Silvina-García-Larraburu-147895613528/",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "nora.gimenez@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/507.gif",
    "nombre": "Giménez, Nora del Valle",
    "partido": "Frente de Todos",
    "provincia": "Salta",
    "redes": [
      "nora.gimenez@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "mcgimeneznavarro@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/562.gif",
    "nombre": "Giménez Navarro, María Celeste",
    "partido": "Alianza Unión por la Patria",
    "provincia": "San Juan",
    "redes": [
      "mcgimeneznavarro@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "martin.goerling@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/560.gif",
    "nombre": "Goerling Lara, Enrique Martin",
    "partido": "Juntos por el Cambio",
    "provincia": "Misiones",
    "redes": [
      "martin.goerling@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "maria.gonzalez@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/477.gif",
    "nombre": "González, María Teresa Margarita",
    "partido": "Alianza Unión por la Patria",
    "provincia": "Formosa",
    "redes": [
      "maria.gonzalez@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "victoria.huala@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/531.gif",
    "nombre": "Huala, María Victoria",
    "partido": "Juntos por el Cambio",
    "provincia": "La Pampa",
    "redes": [
      "victoria.huala@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "luis.juez@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/400.gif",
    "nombre": "Juez, Luis Alfredo",
    "partido": "Juntos por el Cambio",
    "provincia": "Córdoba",
    "redes": [
      "luis.juez@senado.gob.ar",
      "https://www.facebook.com/LuisAlfredoJuez",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "mariana.juri@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/534.gif",
    "nombre": "Juri, Mariana",
    "partido": "Frente Cambia Mendoza",
    "provincia": "Mendoza",
    "redes": [
      "mariana.juri@senado.gob.ar",
      "https://twitter.com/mariana_juri",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "alicia.kirchner@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/366.gif",
    "nombre": "Kirchner, Alicia Margarita Antonia",
    "partido": "Alianza Unión por la Patria",
    "provincia": "Santa Cruz",
    "redes": [
      "alicia.kirchner@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "daniel.kroneberger@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/530.gif",
    "nombre": "Kroneberger, Daniel Ricardo",
    "partido": "Juntos por el Cambio",
    "provincia": "La Pampa",
    "redes": [
      "daniel.kroneberger@senado.gob.ar",
      "https://www.facebook.com/kronebergerdaniel",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "sergio.leavy@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/508.gif",
    "nombre": "Leavy, Sergio Napoleón",
    "partido": "Frente de Todos",
    "provincia": "Salta",
    "redes": [
      "sergio.leavy@senado.gob.ar",
      "https://www.facebook.com/ElOsoLeavy",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "claudia.ledesma@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/509.gif",
    "nombre": "Ledesma Abdala de Zamora, Claudia",
    "partido": "Frente Cívico por Santiago",
    "provincia": "Santiago del Estero",
    "redes": [
      "claudia.ledesma@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "marcelo.lewandowski@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/538.gif",
    "nombre": "Lewandowski, Marcelo Nestor",
    "partido": "Frente de Todos",
    "provincia": "Santa Fe",
    "redes": [
      "marcelo.lewandowski@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "carlos.linares@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/529.gif",
    "nombre": "Linares, Carlos Alberto",
    "partido": "Frente de Todos",
    "provincia": "Chubut",
    "redes": [
      "carlos.linares@senado.gob.ar",
      "https://www.facebook.com/LinaresCarlosCR",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "carolina.losada@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/536.gif",
    "nombre": "Losada, Carolina",
    "partido": "Juntos por el Cambio",
    "provincia": "Santa Fe",
    "redes": [
      "carolina.losada@senado.gob.ar",
      "https://www.facebook.com/carolosada",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "martin.lousteau@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/497.gif",
    "nombre": "Lousteau, Martín",
    "partido": "Juntos por el Cambio",
    "provincia": "Ciudad Autónoma de Buenos Aires",
    "redes": [
      "martin.lousteau@senado.gob.ar",
      "https://www.facebook.com/gugalusto",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "cristina.lopez@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/574.gif",
    "nombre": "López, Cándida Cristina",
    "partido": "Alianza Frente de Todos",
    "provincia": "Tierra del Fuego, Antártida e Islas del Atlántico Sur",
    "redes": [
      "cristina.lopez@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "florencia.lopez@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/555.gif",
    "nombre": "López, María Florencia",
    "partido": "Alianza Unión por la Patria",
    "provincia": "La Rioja",
    "redes": [
      "florencia.lopez@senado.gob.ar",
      "https://www.facebook.com/share/19thWpNnL2/?mibextid=wwXIfr",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "juan.manzur@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/576.gif",
    "nombre": "Manzur, Juan Luis",
    "partido": "Alianza Frente de Todos",
    "provincia": "Tucumán",
    "redes": [
      "juan.manzur@senado.gob.ar",
      "https://www.facebook.com/JuanManzurOficial/",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "mayans@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/294.gif",
    "nombre": "Mayans, José Miguel Ángel",
    "partido": "Alianza Unión por la Patria",
    "provincia": "Formosa",
    "redes": [
      "mayans@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "sandra.mendoza@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/540.gif",
    "nombre": "Mendoza, Sandra Mariela",
    "partido": "Frente de Todos",
    "provincia": "Tucumán",
    "redes": [
      "sandra.mendoza@senado.gob.ar",
      "https://www.facebook.com/sandramentuc",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "maria.moises@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/554.gif",
    "nombre": "Moises, María Carolina",
    "partido": "Alianza Unión por la Patria",
    "provincia": "Jujuy",
    "redes": [
      "maria.moises@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "gerardo.montenegro@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/446.gif",
    "nombre": "Montenegro, Gerardo Antenor",
    "partido": "Frente de Todos",
    "provincia": "Santiago del Estero",
    "redes": [
      "gerardo.montenegro@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "jose.neder@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/510.gif",
    "nombre": "Neder, José Emilio",
    "partido": "Frente Cívico por Santiago",
    "provincia": "Santiago del Estero",
    "redes": [
      "jose.neder@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "stella.olalla@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/503.gif",
    "nombre": "Olalla, Stella Maris",
    "partido": "Juntos por el Cambio",
    "provincia": "Entre Ríos",
    "redes": [
      "stella.olalla@senado.gob.ar",
      "https://www.facebook.com/senadorastellaolalla",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "bruno.olivera@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/563.gif",
    "nombre": "Olivera Lucero, Bruno Antonio",
    "partido": "Alianza la Libertad Avanza",
    "provincia": "San Juan",
    "redes": [
      "bruno.olivera@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "juan.pagotto@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/557.gif",
    "nombre": "Pagotto, Juan Carlos",
    "partido": "Alianza la Libertad Avanza",
    "provincia": "La Rioja",
    "redes": [
      "juan.pagotto@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "francisco.paoltroni@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/551.gif",
    "nombre": "Paoltroni, Francisco Manuel",
    "partido": "Alianza la Libertad Avanza",
    "provincia": "Formosa",
    "redes": [
      "francisco.paoltroni@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "oscar.parrilli@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/504.gif",
    "nombre": "Parrilli, Oscar Isidro",
    "partido": "Frente de Todos",
    "provincia": "Neuquén",
    "redes": [
      "oscar.parrilli@senado.gob.ar",
      "https://www.facebook.com/oscarparrillineuquen/",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "pilatti@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/435.gif",
    "nombre": "Pilatti Vergara, María Inés",
    "partido": "Frente de Todos",
    "provincia": "Chaco",
    "redes": [
      "pilatti@senado.gob.ar",
      "https://twitter.com/pilattivergara",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "mariano.recalde@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/498.gif",
    "nombre": "Recalde, Mariano",
    "partido": "Frente de Todos",
    "provincia": "Ciudad Autónoma de Buenos Aires",
    "redes": [
      "mariano.recalde@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "jesus.rejal@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/556.gif",
    "nombre": "Rejal, Jesús Fernando",
    "partido": "Alianza Unión por la Patria",
    "provincia": "La Rioja",
    "redes": [
      "jesus.rejal@senado.gob.ar",
      "https://www.facebook.com/frejal?locale=es_LA",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "antonio.rodas@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/500.gif",
    "nombre": "Rodas, Antonio José",
    "partido": "Frente de Todos",
    "provincia": "Chaco",
    "redes": [
      "antonio.rodas@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "sonia.rojas@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/559.gif",
    "nombre": "Rojas Decut, Sonia Elizabeth",
    "partido": "Frente Renovador de la Concordia-Innovación Federal",
    "provincia": "Misiones",
    "redes": [
      "sonia.rojas@senado.gob.ar",
      "https://www.facebook.com/100087662746565",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "juan.romero@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/35.gif",
    "nombre": "Romero, Juan Carlos",
    "partido": "Juntos por el Cambio",
    "provincia": "Salta",
    "redes": [
      "juan.romero@senado.gob.ar",
      "https://www.facebook.com/romeroxsalta",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "fernando.salino@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/566.gif",
    "nombre": "Salino, Fernando Aldo",
    "partido": "Alianza Unión por la Patria",
    "provincia": "San Luis",
    "redes": [
      "fernando.salino@senado.gob.ar",
      "https://www.facebook.com/fersalino",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "silvia.sapag@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/251.gif",
    "nombre": "Sapag, Silvia Estela",
    "partido": "Frente de Todos",
    "provincia": "Neuquén",
    "redes": [
      "silvia.sapag@senado.gob.ar",
      "https://twitter.com/sapag_silvia",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "monica.silva@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/572.gif",
    "nombre": "Silva, Mónica Esther",
    "partido": "Juntos Somos Río Negro",
    "provincia": "Río Negro",
    "redes": [
      "monica.silva@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "rodolfo.suarez@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/571.gif",
    "nombre": "Suárez, Rodolfo Alejandro",
    "partido": "Frente Cambia Mendoza",
    "provincia": "Mendoza",
    "redes": [
      "rodolfo.suarez@senado.gob.ar",
      "https://www.facebook.com/RodolfoSuarezMendoza",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "guadalupe.tagliaferri@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/499.gif",
    "nombre": "Tagliaferri, Guadalupe",
    "partido": "Juntos por el Cambio",
    "provincia": "Ciudad Autónoma de Buenos Aires",
    "redes": [
      "guadalupe.tagliaferri@senado.gob.ar",
      "https://www.facebook.com/lupetagliaferri",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "edith.terenzi@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/528.gif",
    "nombre": "Terenzi, Edith Elizabeth",
    "partido": "Juntos por el Cambio Chubut",
    "provincia": "Chubut",
    "redes": [
      "edith.terenzi@senado.gob.ar",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "sergio.unac@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/561.gif",
    "nombre": "Uñac, Sergio Mauricio",
    "partido": "Alianza Unión por la Patria",
    "provincia": "San Juan",
    "redes": [
      "sergio.unac@senado.gob.ar",
      "https://www.facebook.com/sergio.unac",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "mercedes.valenzuela@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/525.gif",
    "nombre": "Valenzuela, Mercedes Gabriela",
    "partido": "Eco + Vamos Corrientes",
    "provincia": "Corrientes",
    "redes": [
      "mercedes.valenzuela@senado.gob.ar",
      "https://www.facebook.com/gabivalenzuela.ok",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "alejandra.vigo@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/523.gif",
    "nombre": "Vigo, Alejandra María",
    "partido": "Hacemos por Córdoba",
    "provincia": "Córdoba",
    "redes": [
      "alejandra.vigo@senado.gob.ar",
      "https://www.facebook.com/VigoAlejandra",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "eduardo.vischi@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/524.gif",
    "nombre": "Vischi, Eduardo Alejandro",
    "partido": "Eco + Vamos Corrientes",
    "provincia": "Corrientes",
    "redes": [
      "eduardo.vischi@senado.gob.ar",
      "https://www.facebook.com/PetecoVischi/",
    ],
    "telefono": "(+54 11) 28223000",
  },
  {
    "email": "victor.zimmermann@senado.gob.ar",
    "foto": "https://www.senado.gob.ar/bundles/senadosenadores/images/fsena/501.gif",
    "nombre": "Zimmermann, Victor",
    "partido": "Juntos por el Cambio",
    "provincia": "Chaco",
    "redes": [
      "victor.zimmermann@senado.gob.ar",
      "https://www.facebook.com/VictorZimmermannOK",
    ],
    "telefono": "(+54 11) 28223000",
  },
]
`;
